mne/io/kit/tests/test_coreg.py

```diff
@@ -28,3 +30,16 @@ def test_io_mrk():
     _write_dig_points(path, pts)
     pts_2 = read_mrk(path)
     assert_array_equal(pts, pts_2, "read/write mrk to text")
+
+    # pickle
+    fname = os.path.join(tempdir, 'mrk.pickled')
+    with open(fname, 'wb') as fid:
+        pickle.dump({'mrk': pts}, fid)
+    pts_2 = read_mrk(fname)
+    assert_array_equal(pts_2, pts, "pickle mrk")
+    with open(fname, 'wb') as fid:
+        pickle.dump({}, fid)
```

Comment:
I'm not sure where the convention comes from. I think it may because it's indeed more explicit to those who don't speak python, but I'm not sure.